azext_iot/central/_help.py

```diff
@@ -550,6 +708,25 @@ def _load_central_command_help():
                 -i {interfaceid}
                 --cn {commandname}
                 -k {payload}
+
+            - name: Run component command response
```

Comment:
what does run command response mean?

Reply:
I used the previous command name. I'll change them since they're not very readable.